spec/index.bs

```diff
@@ -3792,7 +3796,7 @@ dictionary GPUTextureDescriptor
 
         Formats in this list must be [=texture view format compatible=] with the texture format.
 
-        <div algorithm>
```

Comment:
nit: I noticed you added text to most algorithms specifying which timeline they run on, but not several of them including this one. Looks like mostly ones where there wasn't a good spot to specify it? I don't have a strong opinion about being totally consistent about this though so not a blocker (you can always figure out what timeline you're on from the caller if you really need to).

Reply:
Yeah, a few of them didn't have a great spot to list it. I did want to revisit them, though, and maybe do some larger refactoring to help normalize them a bit.

This one in particular, after your above comment, feels like it should actually be a `const` timeline algorithm, because all of the values it touches are passed to it directly, meaning that presumably you could call it from any timeline as long as you were able to get those values. I've been marking some of the algorithms I had left un-timelined before as `const` while addressing your feedback, as you suggested, but it does leave a question of how to refer to them in prose. "run the following steps from [=any timeline=]?" (`any timeline` can be a alt link to the general timelines section)

Reply:
Maybe we could make the visible wording for data-timeline=const be "timeline-agnostic" as in "timeline-agnostic steps"?

Reply:
Oh, I like that! Gonna revise some of my changes to use that phrasing.